src/static/css/pages/bureau.less

```diff
@@ -6,33 +6,15 @@
 // Size of play button image;
 @play-btn-size__px: 73px;
 
-.bureau-mission {
-    margin-bottom: unit(45px / @base-font-size-px, em);
-
-    .respond-to-min(1024px, {
-        margin-top: unit(60px / @base-font-size-px, em);
-    });
-
-    &_media .image-container {
-        margin-left: 10px;
-    }
-
-    &_section {
-        margin-top: unit(30px / @base-font-size-px, em);
-    }
-
-    &_section + &_section {
-        .respond-to-min(600px, {
-            border-top: 1px solid @gray-50;
-            padding-top: unit(30px / @base-font-size-px, em);
-        });
-    }
-
-    .media_image {
-        width: 120px;
-        .respond-to-max(1024px, {
-            width: 80px;
-        });
+// Override general slick carousel overrides defined in mobile-carousel.less.
+// Removes bottom border, because the-bureau has a bordered block below
+// its carousel.
+.the-bureau {
```

Comment:
Sorry, I meant you can write them on a single line since there's nothing else inside. Makes for a quicker scanning when you're trying to find matching selectors from the browser.